dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb

```diff
@@ -8,6 +8,14 @@ def new_form
       )
     end
 
+    def create
+      # Ignore duplicate submissions. The UI should not allow this to happen anyway
+      existing_form = Pd::Application::Facilitator1819Application.find_by(user: current_user)
+      return render json: {id: existing_form.id}, status: :ok if existing_form
```

Comment:
You can one-line this with 
`
return render json: {id: existing_form.id}, status: :ok if Pd::Application::Facilitator1819Application.exists?(user: current_user)
`